nuxeo-services/login/nuxeo-platform-login-openid/src/test/resources/OSGI-INF/openid-provider-contrib.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.oauth2.openid.provider.test">
+
+  <extension target="org.nuxeo.ecm.platform.oauth2.openid.OpenIDConnectProviderRegistry" point="providers">
+
+      <provider>
+
+              <name>MY_NAME</name>
```

Comment:
You probably don't need to have every other line be an empty line.

Reply:
Done (removed empty lines)